src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatController.ts

```diff
@@ -327,7 +329,10 @@ export class TerminalChatController extends Disposable implements ITerminalContr
 				this._chatAccessibilityService.acceptResponse(responseContent, accessibilityRequestId);
 				const containsCode = responseContent.includes('```');
```

Comment:
note we still need `containsCode` check above bc have to call `updateChatMessage` for the code blocks to get rendered